src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: 166px;
+  height: 358px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  color: $main-text-color;
+  padding: 32px 16px 16px;
+
+  display: flex;
+  flex-direction: column;
+}
+
+.card__name {
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 18px;
+  text-decoration: none;
+  color: $main-text-color;
+  margin-top: 40px;
+}
+
+.card__product {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: $second-text-color;
+  margin-top: 4px;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 16px;
+}
+
+.reviews {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-decoration: none;
+  color: $main-text-color;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+}
+
+.price__name {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: $second-text-color;
+}
+
+.price__number {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+}
+
+.card__button {
+  width: 166px;
```

Comment:
The width of `.card__button` is explicitly set, which might not be necessary if it is supposed to match the card width. Consider using `width: 100%;` to ensure the button width is always equal to the card width, which would be more maintainable if the card width changes in the future.